src/components/ChatEntry.js

```diff
@@ -1,22 +1,43 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp'
 
 const ChatEntry = (props) => {
+
+
+  const updateLikeButtonClick = () =>{
+    const updateChat ={
+      id : props.id,
+      sender : props.sender,
+      body : props.body,
+      liked : !props.liked, 
+      timeStamp : props.timeStamp,
+    }
+    props.onUpdate(updateChat);
+  }
+
+  const heartColor = props.liked ? '❤️': '🤍'; 
+
   return (
     <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+      <h2 className="entry-name">{props.sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p> {props.body} </p>
+        <p className="entry-time"><TimeStamp time = {props.timeStamp}></TimeStamp></p>
```

Comment:
👍 Nice use of the TimeStamp component.

Stylistically, prefer to omit the spaces around `=` in JSX attributes. And JSX prefers using the `/>` syntax for closing empty components.

```js
        <p className="entry-time"><TimeStamp time={props.timeStamp} /></p>
```